src/main/java/uk/gov/hmcts/cmc/claimstore/events/paidinfull/PaidInFullCitizenNotificationHandler.java

```diff
@@ -3,31 +3,38 @@
 import com.google.common.collect.ImmutableMap;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.appinsights.AppInsights;
+import uk.gov.hmcts.cmc.claimstore.appinsights.AppInsightsEvent;
 import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
 import uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationService;
 import uk.gov.hmcts.cmc.domain.models.Claim;
 
 import java.util.Map;
 
+import static uk.gov.hmcts.cmc.claimstore.appinsights.AppInsights.REFERENCE_NUMBER;
 import static uk.gov.hmcts.cmc.claimstore.services.notifications.NotificationReferenceBuilder.PaidInFull.referenceForDefendant;
 
 @Component
 public class PaidInFullCitizenNotificationHandler {
 
     private final NotificationService notificationService;
     private final NotificationsProperties notificationsProperties;
+    private final AppInsights appInsights;
 
     public PaidInFullCitizenNotificationHandler(
         NotificationService notificationService,
-        NotificationsProperties notificationsProperties
+        NotificationsProperties notificationsProperties,
+        AppInsights appInsights
     ) {
         this.notificationService = notificationService;
         this.notificationsProperties = notificationsProperties;
+        this.appInsights = appInsights;
     }
 
     @EventListener
     public void onPaidInFullEvent(PaidInFullEvent event) {
         this.notifyDefendantForPaidInFull(event);
+        this.paidInFullAppInsight(event);
```

Comment:
To be consistent with existing code, we should fire this event in this method of `ClaimService`
`uk.gov.hmcts.cmc.claimstore.services.ClaimService#paidInFull`

Please revert all the changes in this file and add the below line in `ClaimService.paidInFull(...)` after line 335.
```
appInsights.trackEvent(AppInsightsEvent.PAID_IN_FULL, REFERENCE_NUMBER, claim.getReferenceNumber());
```

Reply:
Will do cheers!